packages/prover/test/unit/utils/conversion.test.ts

```diff
@@ -0,0 +1,90 @@
+import {expect} from "chai";
+import {chunkIntoN} from "../../../src/utils/conversion.js";
+
+describe("utils/conversion", () => {
+  describe("chunkIntoN", () => {
+    const testCases = [
+      {
+        title: "when even number of chunks",
```

Comment:
what about just

```suggestion
        title: "even number of chunks",
```
You can keep your preferred phrasing, just wondering what would be the best test descriptions for such cases in general in terms of readability